src/components/main/index.tsx

```diff
@@ -1,12 +1,20 @@
 import React, { FunctionComponent } from "react";
 import { Box, useTheme } from "@chakra-ui/react";
-
-import { Button } from "@components";
+import { Auth } from "aws-amplify";
+import { AuthButton } from "@components";
 
 interface Props {
     title?: string;
 }
 
+async function signOut() {
+    try {
+        await Auth.signOut();
```

Comment:
add the logic to sign out of google properly here (maybe cognito supports this out of the box but you should check)